src/interfaces/vault.ts

```diff
@@ -503,17 +503,18 @@ export class VaultInterface<T extends ChainId> extends ServiceInterface<T> {
     if (overrides.vaultIconOverride) {
       dynamic.metadata.displayIcon = overrides.vaultIconOverride;
     }
-    if (overrides.apyOverride) {
+    if (overrides.apyTypeOverride) {
       if (!dynamic.metadata.apy) {
         dynamic.metadata.apy = this.makeEmptyApy();
       }
-      dynamic.metadata.apy.net_apy = overrides.apyOverride;
+      dynamic.metadata.apy.type = overrides.apyTypeOverride;
     }
-    if (overrides.apyTypeOverride) {
+    if (overrides.apyOverride) {
       if (!dynamic.metadata.apy) {
         dynamic.metadata.apy = this.makeEmptyApy();
       }
-      dynamic.metadata.apy.type = overrides.apyTypeOverride;
+      dynamic.metadata.apy.net_apy = overrides.apyOverride;
+      dynamic.metadata.apy.type = "override";
```

Comment:
@jstashh , do you prefer to use a constant for this? is  there a good file to put this string rather than just a inline plain string?

Reply:
this is fine, we need to add a constants file to the sdk